admin/views/school/select-pupils-not-taking-check.ejs

```diff
@@ -35,37 +35,95 @@
+                        <li>
+                            <div><strong>Absent</strong></div>
+                            <div>Pupil has not taken the check due to absence</div>
+                        </li>
+                        <li>
+                            <div><strong>Left school</strong></div>
+                            <div>Pupil has left the school</div>
+                        </li>
+                        <li>
+                            <div><strong>Incorrect registration</strong></div>
+                            <div>Pupil has been incorrectly added to the Pupil Register. They may have been added by the school, or incorrectly included in the pre-populated list</div>
+                        </li>
+                        <li>
+                            <div><strong>Withdrawn</strong></div>
+                            <div>Pupil has been withdrawn from the check.</div>
+                            <div>This includes pupils who:</div>
+                            <ul>
+                                <li>have just arrived and you have not had time to assess their abilities</li>
+                                <li>have no understanding of multiplication tables</li>
+                                <li>are unable to access the check</li>
+                            </ul>
+                        </li>
+                    </ul>
+                </details>
+            </div>
+
+            <br class="clearfix"/>
+            <br class="clearfix"/>
+
+            <h2>2. Select pupil(s)</h2>
+            <% if (pupilsList && pupilsList.length > 0) { %>
+            <table class="spacious sorting" id="pupilsList" role="listbox" aria-label="Select pupils.">
+                <thead>
+                <tr>
+                    <th scope="col">
+                        <a href="/school/pupils-not-taking-check/select-pupils/name/<%= htmlSortDirection.name %>">Pupil name <span class="<%= arrowSortDirection.name %>"><span>Sort by pupil name <%= htmlSortDirection.name %></span></span></a>
+                    </th>
+                    <th scope="col">
+                        <a href="/school/pupils-not-taking-check/select-pupils/reason/<%= htmlSortDirection.reason %>">Reason <span class="<%= arrowSortDirection.reason %>"><span>Sort by reason <%= htmlSortDirection.reason %></span></span></a>
+                    </th>
+                    <th scope="col">
+                        <div class="tick-all-checkboxes-wrapper">
+                            <label class="tick-all-checkboxes" id="selectAll" for="tickAllCheckboxes">Select all</label>
+                            <label class="tick-all-checkboxes all-hide" id="unselectAll" for="tickAllCheckboxes">Unselect all</label>
+                            <div class="multiple-choice-mtc">
+                                <input id="tickAllCheckboxes" name="allPupils" type="checkbox" aria-label="Select all pupils.">
+                                <div></div>
```

Comment:
Should not need this empty div

Reply:
That empty div is actually needed, it's replacing the role of the label to create the "tick" visual effect.
This code is a modification of the toolkit.

Reply:
No, we shouldn't.

Another question - does the `all-hide` class remove the second label from screen-readers?

Reply:
it does, that's the goal

Reply:
OK good